### ISSUE TITLE
check_predictions() produce empty plot

### ISSUE THREAD

**Issue by tamsa42** [easystats/performance#587]
I was checking a binomial glm using both check_model and check_prediction functions, and in case I include any fixed factor in the model, the prediction plot is empty. 
![image](https://github.com/easystats/performance/assets/77250541/690a558e-2bc1-4ce3-9adf-bad44bcae956)
 if I add only continuous explanatory variables to the model, the plot looks fine.  
![image](https://github.com/easystats/performance/assets/77250541/6ca97387-51cd-4acc-b9fe-2119bc5bf18a) there is no error message or such. tried to google the issue but had no luck. I use a fresh install of R and R studio with fresh packages, but I had the same experience with R 4.2.2 too

**Comment by strengejacke** [comment]
do you have a reproducible example?

**Comment by tamsa42** [comment]
here it is. but it is even weirder now, as I cannot even reproduce the figure I produced earlier, so it is not working with any of the covariates or the factor I included in the sample. 

[sample.csv](https://github.com/easystats/performance/files/11478707/sample.csv)

an example script what I do: 

`sample <- read.csv("sample.csv")
mod <- glm(resp ~ fact + covar1 + covar2, family = binomial(link = "logit"), data=sample)
check_model(mod)`

![image](https://github.com/easystats/performance/assets/77250541/f92ffa41-0a9c-4c51-85b8-c0a7dc122b29)

the VIF plot also looks suspicious...

but based on the summary, the model looks fine. there are some missing values, but the issue is the same after excluding those lines.

**Comment by strengejacke** [comment]
Should be fixed in #588. You need to change the `bandwidth` argument for the density estimation.

``` r
library(performance)
sample <- read.csv("d:/Downloads/sample.csv")
mod <- glm(resp ~ fact + covar1 + covar2, family = binomial(link = "logit"), data=sample)
check_predictions(mod, bandwidth = "nrd0")
```

![](https://i.imgur.com/FElqk7W.png)

<sup>Created on 2023-05-15 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>
